### PR TITLE
Add sunrise color scheme

### PR DESCRIPTION
**Pull request by mreishus**
One part of #47036

#### Changes proposed in this Pull Request

* Add sunrise color scheme

#### Testing instructions

* Visit /me/account
* Choose sunrise and save

To see the wp-admin color scheme, create an atomic site, then visit /wp-admin/profile.php, choose color scheme and save.

Calypso:

Wp-Admin:

#### Notes

If you're following along with Ectoplasm from #47122, you'll see I generated a palette of "Ectoplasm green" in -10 to -90 values.  For this one, I found that "Studio Orange" was a close enough match for primary+accent colors and reused it.  The hue of the highlight color in the wp-admin scheme is 25, and studio orange is a gradient between 25 and 35.  The sidebar and button colors should still match exactly, though.

### CODE REVIEW

File: packages/calypso-color-schemes/src/shared/color-schemes/_sunrise.scss [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    --theme-text-color-rgb: 255, 255, 255; /* Manual conversion */
+    --theme-base-color: #cf4944; /* Direct from wp-admin */
+    --theme-base-color-rgb: 207, 73, 68; /* Manually computed https://www.rapidtables.com/convert/color/hex-to-rgb.html */
+    --theme-submenu-background-color: #be3631; /* darken( $base-color, 7% ), computed: http://scg.ar-ch.org/
+    --theme-icon-color: #f3f1f1; /* Direct from wp-admin */
+    --theme-highlight-color: #dd823b; /* Direct from wp-admin */
+    --theme-highlight-color-rgb: 221, 130, 59; /* Manually computed https://www.rapidtables.com/convert/color/hex-to-rgb.html */
+    --theme-notification-color: #ccaf0b; /* Direct from wp-admin */
+
+}
```
Comment (cpapazoglou) [inline packages/calypso-color-schemes/src/shared/color-schemes/_sunrise.scss:38]: As also suggested in https://github.com/Automattic/wp-calypso/pull/47122 let's try consolidating these variables in _defaut.scss.

**Review by cpapazoglou — CHANGES_REQUESTED** [review]
1. The unread dot needs some treatment


2. Sidebar Menu Heading should be white


3. Section Headers are not legible

File: packages/calypso-color-schemes/src/shared/color-schemes/_sunrise.scss (hunk 1 above)
Comment (mreishus) [reply packages/calypso-color-schemes/src/shared/color-schemes/_sunrise.scss]: Done

**Comment by mreishus** [comment]
I believe all of the issues mentioned above are fixed.

**Review by cpapazoglou — APPROVED** [review]
Thanks for pushing the changes @mreishus!

LGTM!

**Pull request merged**